applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -664,16 +710,28 @@ static int initialize(le_audio_receive_cb recv_cb, le_audio_timestamp_cb timestm
 		return -ECANCELED;
 	}
 #else
-	ret = bt_pacs_set_available_contexts(
-		BT_AUDIO_DIR_SINK, BT_AUDIO_CONTEXT_TYPE_MEDIA | BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED);
+
+	ret = bt_pacs_set_supported_contexts(BT_AUDIO_DIR_SINK,
+				     BT_AUDIO_CONTEXT_TYPE_MEDIA |
+				     BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED);
+
+	if (ret) {
+		LOG_ERR("Supported context set failed. Err: %d ", ret);
+		return ret;
+	}
+
+	ret = bt_pacs_set_available_contexts(BT_AUDIO_DIR_SINK,
+				     BT_AUDIO_CONTEXT_TYPE_MEDIA |
+				     BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED);
+
 	if (ret) {
-		LOG_ERR("Available context set failed");
+		LOG_ERR("Available context set failed. Err: %d", ret);
 		return ret;
 	}
 #endif /* CONFIG_STREAM_BIDIRECTIONAL */
 
 	for (int i = 0; i < ARRAY_SIZE(audio_streams); i++) {
-		bt_audio_stream_cb_register(&audio_streams[i], &stream_ops);
+			bt_bap_stream_cb_register(&audio_streams[i], &stream_ops);
```

Comment:
Seems to be an extra tab here? Might just be github

Reply:
Hmmm, several of the files are not clang-formatted

Reply:
I have gone through all files and clang-formatted now